lint-staged.config.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const micromatch = require('micromatch');
```

Comment:
why do we need this file? I'm generally not in favor of adding more boilerplate code to have to maintain when the configuration in package.json has been sufficient thus far

Reply:
Yeh this is the only way to have prettier run on all the format it supports (`prettier
  .getSupportInfo()`) without hardcoding a static list of extensions.

Reply:
I would feel a lot better about a static list of extensions.